template/index.js

```diff
@@ -1,5 +1,5 @@
 import { AppRegistry } from 'react-native';
 import App from './src/App';
-import { name as appName } from './app.json';
+import { name as appName } from './app.json.js';
```

Comment:
curious what the background was on this one?

Reply:
Um, that's strange...I didn't do that? (intentionally at least)

Reply:
Fixed